gravityflow/gravityflowwoocommerce
# Remove unnecessary alert when enabling feed setting

## Description
[Issue#43](https://github.com/gravityflow/backlog/issues/43)
Unnecessary alert when enabling Create Entry on WooCommerce Feed Settings.

## Testing instructions
1. Enable 'Create Entry' in WooCommerce Feed Settings with the master, the confirmation alert appears.
2. Switch to this branch and repeat step 1. Confirm the confirmation alert does not appear now.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
